packages/epo-react-lib/src/layout/MasonryGrid/MasonryGrid.tsx

```diff
@@ -0,0 +1,55 @@
+import * as Styled from "./styles";
+import Tile from "./MasonryGridTile";
+import { FunctionComponent } from "react";
+import { GalleryEntry, GalleryItemCategory } from "@/types/gallery";
+
+interface MasonryGridProps {
+  items: GalleryEntry[];
+  randomize?: boolean;
+}
+
+const MasonryGrid: FunctionComponent<MasonryGridProps> = ({
+  items,
+  randomize,
+}) => {
+  const template = (i: number, width: number) => `
+          a:nth-child(${i + 1}n + ${i + 1}) {
+            width: ${width}%;
+          }
+        `;
+
+  const getBrickSizes = () => {
+    const widthMap = [20, 20, 20, 20, 30, 30, 30, 40, 40, 80];
+    let str = "";
+    for (let i = 0; i < 20; i++) {
+      const width = randomize
```

Comment:
the random functionality was included but commented out, I opted to add it in but default off